api/src/main/java/org/pmiops/workbench/db/dao/RdrExportDao.java

```diff
@@ -12,7 +12,7 @@
       nativeQuery = true,
       value =
           "select u.user_id from user u LEFT JOIN rdr_export rdr on"
-              + " u.user_id = rdr.export_id and rdr.entity_type = 1 where "
+              + " u.user_id = rdr.export_id and rdr.entity_type = 1 where u.last_modified_time is null or "
```

Comment:
are we ok with hitting the RDR multiple times with the same user account? As is, I believe a user will keep getting sent until they log in and their lat_modified_time gets set.

Reply:
Actually i am going to push change to set the  value of  last_modified_date time as either today's date or first_sign in just so it is picked  only the first time